scripts/linux/pkg/deb/assets/build-deb.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash 
+
+shopt -s lastpipe
+set -e
+
+PACKAGE_REVISION="${PACKAGE_REVISION:-1}"
+DOWNLOAD_URL=https://daily-builds.s3.amazonaws.com/RavenDB-$RAVENDB_VERSION-linux-x64.tar.bz2 
+
+if [ -z $RAVENDB_VERSION ]; then
+    echo "RAVENDB_VERSION is not set."
+    exit 1
+fi
+
+echo "Building a DEB package for RavenDB $RAVENDB_VERSION in $BUILD_DIR"
+
+if [ -d $BUILD_DIR ]; then
+    pushd $BUILD_DIR && rm -rf *
+    trap popd EXIT
+else
+    echo "BUILD_DIR does not exist."
+    exit 1 
+fi
+
+tarBall="/cache/ravendb-${RAVENDB_VERSION}.tar.bz2"
+
+mkdir -p "$DEBPKG-$RAVENDB_VERSION"
+
+if [ ! -f "$tarBall" ]; then
+    wget -O $tarBall $DOWNLOAD_URL
+fi
+
+tar xjf $tarBall -C $BUILD_DIR
+
+python3 - <<SCRIPT || exit 1
+import json
+import codecs
+
+defaultSettingsPath = '$BUILD_DIR/RavenDB/Server/settings.default.json'
+with codecs.open(defaultSettingsPath, 'r', 'utf-8-sig') as f:
+    settings = json.load(f);
+    
+del settings['DataDir']
```

Comment:
I think we should default to `/data/ravendb` and create that in the install phase

Reply:
We do that, create /var/lib/ravendb/data